cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/BasicMapReduceContext.java

```diff
@@ -81,8 +83,7 @@
   private final Map<String, LocalizeResource> resourcesToLocalize;
   private final Supplier<MapReduceTaskContextProvider> taskContextProviderSupplier;
 
-  private InputFormatProvider inputFormatProvider;
-
+  private Map<String, MapperInput> inputs;
```

Comment:
it would make sense to explain what the key means. Is it the name of  the input?